src/client/app/views/home/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+import { Container } from "reactstrap";
+
+export default class ModList extends Component<{ history: any; user: any | null }> {
```

Comment:
This should probably be called Home or something instead of ModList. I'm assuming that's just from copy-paste.

Reply:
Oof. You got me there. Gonna change that with the other small fixes.